src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        Colors[] colors = Colors.values();
+        Colors color = colors[random.nextInt(5)];
```

Comment:
`5` is a magic number
```suggestion
        Colors color = colors[random.nextInt(colors.length)];
```

Reply:
not fixed